src/interfaces/records/messages/records-write.ts

```diff
@@ -28,14 +28,18 @@ export type RecordsWriteOptions = AuthCreateOptions & {
   published?: boolean;
   datePublished?: string;
   dataFormat: string;
+  authorizationSignatureInput: SignatureInput;
+  attestationSignatureInputs?: SignatureInput[];
```

Comment:
I find it strange to have an optional type over an array. Is this standard practice in TS ?

Reply:
This tries to make dev experience extra nicer, so that when they don't care about attestation, they are don't have to give and empty array just to make the API happy.